File: tests/fixtures/python/sample.py
import os
import sys as system
from pathlib import Path

class FileProcessor:
    def process(self, path):
        return os.path.exists(path)

    def validate(self, path):
        return Path(path).is_file()

def read_file(path):
    with open(path) as f:
        return f.read()

File: tests/fixtures/rust/sample.rs
use std::path::Path;

struct FileReader {
    path: String,
}

trait Readable {
    fn read(&self) -> String;
}

impl FileReader {
    fn new(path: String) -> Self {
        FileReader { path }
    }

    fn exists(&self) -> bool {
        Path::new(&self.path).exists()
    }
}
